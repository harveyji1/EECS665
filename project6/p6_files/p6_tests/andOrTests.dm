main : () bool{
    myFalse : bool = false;
    myTrue : bool = true;
    myAnd : bool = myTrue and myFalse;
    myOr : bool = myTrue or myFalse;
}