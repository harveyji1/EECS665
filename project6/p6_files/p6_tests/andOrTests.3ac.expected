[BEGIN GLOBALS]
main
[END GLOBALS]
[BEGIN main LOCALS]
myFalse (local var of 8 bytes)
myTrue (local var of 8 bytes)
myAnd (local var of 8 bytes)
myOr (local var of 8 bytes)
[END main LOCALS]
main:       enter main
lbl_0:      leave main